docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/01 - Identifiers.md

```diff
@@ -0,0 +1,69 @@
+# Identifiers
+
+## Overview
+By default LORIS provides 2 different identifiers for each participant or subject: 
+
+- **CandID**, also known as the **DCCID**, is a unique randomized 6-digit numeric ID (e.g. '436792').  It is completely anonymous, and is assigned automatically by Loris upon participant registration. The CandID is not configurable.
```

Comment:
```suggestion
- **CandID**, also known as the **DCCID**, is a unique randomized 6-digit numeric ID (e.g. '436792') assigned automatically by LORIS upon participant registration. The CandID is not configurable.
```

I don't know if it really makes sense to refer to this as being anonymous. I kind of know what you mean but I think it's also confusing

Reply:
@johnsaigle I'm not the one who wrote it so if it makes sense to @driusan and/or @christinerogers I dont mind accepting the suggestion

Reply:
I don't know what your question is.. it's removing one clause of the sentence, so it doesn't make it any more or less accurate.